media-gfx/gimp/files/gimp-2.10_fix_configure_GCC13_implicit_function_declarations.patch

```diff
@@ -0,0 +1,24 @@
+Gentoo issue: https://bugs.gentoo.org/899796
```

Comment:
Please send this upstream?

Reply:
(for the old branch)

Reply:
I have problems with pushing to fork via ssh key so I create issue and attached patch:
https://gitlab.gnome.org/GNOME/gimp/-/issues/9467